<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
color: rgb(0, 0, 0);
background-color: rgb(168, 171, 171);
border-style: solid

</string>
    </property>
    <property name="title">
     <string>SignIn</string>
    </property>
    <widget class="QPushButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>311</width>
       <height>32</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 2px;
background-color: rgb(90, 148, 255);
border-style: solid;
border-color: darkblue;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>311</width>
       <height>88</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-color: rgb(56, 255, 46);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="styleSheet">
           <string notr="true">border-color: rgb(0, 255, 19);
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>pushButton_login</zorder>
   </widget>
   <widget class="QLabel" name="label_image1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rrsources/img/Bmccfrontpage.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Bmcc Resources</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>-20</y>
      <width>891</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 600 14pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>  welcome to BMCC resources , Your personalized gateway to an enriched college experiance! </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 600 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>  Say GoodBye to uncertainty and hello to thriving college journey. Let’s embark on this exciting adventure together.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 13pt &quot;American Typewriter&quot;;</string>
    </property>
    <property name="text">
     <string>  Let’s embark on this exciting adventure together.</string>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_image1</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
